src/components/clusters/ClusterDetailsMultiRegion/components/AccessRequest/components/AccessRequestTable.tsx

```diff
@@ -54,7 +61,17 @@ const AccessRequestTable = ({
         <AccessRequestStateIcon accessRequest={accessRequestItem} />
       </Td>
       <Td dataLabel={columnsNames.id.title}>{accessRequestItem.id}</Td>
-      <Td dataLabel={columnsNames.created_at.title}>{accessRequestItem.created_at}</Td>
+      {/* <Td dataLabel={columnsNames.created_at.title}>{accessRequestItem.created_at}</Td> */}
+      <Td dataLabel={columnsNames.created_at.title}>
+        <Timestamp
+          date={new Date(accessRequestItem.created_at || '')}
+          dateFormat={TimestampFormat.short}
+          timeFormat={TimestampFormat.long}
+          shouldDisplayUTC
+          is12Hour={false}
```

Comment:
is12Hour={false}, so we're showing 24hr formats for these?
Note first, and the last, <Timestamp..>s in this PR do not have this attribute/property.